auth_test.go

```diff
@@ -0,0 +1,36 @@
+package passage_test
+
+import (
+	"os"
+	"sync"
+	"testing"
+
+	"github.com/joho/godotenv"
+	"github.com/passageidentity/passage-go"
+	"github.com/stretchr/testify/require"
+)
+
+// should be run with the -race flag, i.e. `go test -race -run TestAppJWKSCacheWriteConcurrency`
+func TestAppJWKSCacheWriteConcurrency(t *testing.T) {
+	err := godotenv.Load(".env")
+	require.Nil(t, err)
+
+	appID := os.Getenv("PASSAGE_APP_ID")
+	apiKey := os.Getenv("PASSAGE_API_KEY")
```

Comment:
might want to mock this out at some point so we can drop the dependency on a working app that returns a jwk. then we can drop all the testing secrets in this repo